pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -3060,33 +3060,18 @@ var _ = Describe("Converter", func() {
 		)
 
 		BeforeEach(func() {
-			vmi = kvapi.NewMinimalVMI("testvmi")
+			vmi = libvmi.New(
+				libvmi.WithName("testvmi"),
+				libvmi.WithRng(),
+				libvmi.WithAutoattachMemBalloon(true),
+				libvmi.WithInterface(v1.Interface{Name: "red", Model: "e1000"}),
+				libvmi.WithNetwork(&v1.Network{Name: "red"}),
```

Comment:
You're missing the Pod network. Why not use `*v1.DefaultBridgeNetworkInterface()`/`nonVirtioIface` and `*v1.DefaultPodNetwork()`/`secondaryNetwork` here?

Reply:
I've added in following commit.
```go
libvmi.WithInterface(*v1.DefaultBridgeNetworkInterface()),
libvmi.WithNetwork(v1.DefaultPodNetwork()),
```

Reply:
Can you swap the order? Typically the pod network comes first.